companies/services/report_extractor.py

```
@@ -51,6 +51,28 @@ def extract_content(self, rcept_no: str) -> str | None:
+        Returns:
+            XML 원문 텍스트 (실패 시 None)
+        """
+        try:
+            xml_text = self.dart.document(rcept_no)
+            if xml_text:
+                logger.info(f"보고서 XML 원문 추출 성공: {rcept_no}")
+                return xml_text
+            else:
+                logger.warning(f"보고서 XML 가져오기 실패: {rcept_no}")
+                return None
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)